powershell/make-util.js

```diff
@@ -1,11 +1,11 @@
-
 var admZip = require('adm-zip');
 var child_process = require('child_process');
 var fs = require('fs');
 var path = require('path');
 var process = require('process');
 var shell = require('shelljs');
-var syncRequest = require('sync-request');
+var deasync = require("deasync")
+const Downloader = require("nodejs-file-downloader");
```

Comment:
I don't see if nodejs-file-downloader is present in dependencies.
Seems it was got from the node folder, probably make sense to add it here separately to prevent possible issues